extern/storage-sealing/precommit_policy_test.go

```diff
@@ -112,13 +162,14 @@ func TestBasicPolicyIgnoresExistingScheduleIfExpired(t *testing.T) {
 	exp, err := policy.Expiration(context.Background(), pieces...)
 	require.NoError(t, err)
 
-	assert.Equal(t, 2879, int(exp))
+	assert.Equal(t, 1558079, int(exp))
```

Comment:
I didn't bother showing the math in this (and later) cases as the above units cover those paths. (I did manually check the math on these changes.)